webapp/src/projects.tsx

```diff
@@ -307,7 +307,7 @@ export class ProjectSettingsMenu extends data.Component<ProjectSettingsMenuProps
         const githubUser = !hasIdentity && this.getData("github:user") as UserInfo;
         const reportAbuse = pxt.appTarget.cloud && pxt.appTarget.cloud.sharing && pxt.appTarget.cloud.importing;
         const showDivider = targetTheme.selectLanguage || targetTheme.highContrast || githubUser;
-        const showFeedbackOption = pxt.webConfig.ocvEnabled && targetTheme.feedbackEnabled && targetTheme.ocvFrameUrl && targetTheme.ocvAppId;
+        const showFeedbackOption = pxt.webConfig.ocv?.appId && pxt.webConfig.ocv?.iframeEndpoint;
```

Comment:
Since we do this check, I wonder if it'd be nice to have a little helper function for `ocvEnabled()` somewhere, maybe in the ocv namespace? Not a big deal either way.

Reply:
Ah, yeah, good idea. I definitely found it annoying to type this out every where, I'll add that in.